src/api/contoso_chat/chat_request.py

```diff
@@ -5,7 +5,7 @@
 from sys import argv
 import os
 import pathlib
-from product import product
+from .product import product
```

Comment:
Have you validated this change in your deployment? I am not sure this is the right way to solve this.